src/main/flight/mixer.c

```diff
@@ -759,6 +760,8 @@ void mixTable(void)
             axisPID[PITCH] * currentMixer[i].pitch +
             axisPID[ROLL] * currentMixer[i].roll +
             -mixerConfig->yaw_motor_direction * axisPID[YAW] * currentMixer[i].yaw;
+
+        if (batteryConfig->vbatPidCompensation) motor[i] *= calculateVbatPidCompensation();  // Add voltage compensation
```

Comment:
Yes good point about calculating it once.

But mincommand is not relevant....right? It is never used. The rcCommand throttle starts from min_check and never gets lower than that.